package.json

```diff
@@ -91,7 +91,8 @@
     "ember-template-recast": "^6.0.0",
     "engine.io": "^5.2.1",
     "markdown-it": "^12.3.2",
-    "mout": "^1.2.3"
+    "mout": "^1.2.3",
+    "nanoid": "^3.1.31"
```

Comment:
Why does this appear to added and not updated?

Reply:
Trailing comma (`"mout": "^1.2.3"`  vs `"mout": "^1.2.3",`) triggered an addition.

Reply:
Sorry, I was referring to the fact that nanoid was not in the package.json prior to this PR but it is in the yarn.lock. Just curious.

Reply:
Oops. My bad. Rather than adding multiple resolutions for nanoid in every package.json that has a dependency, surfacing it at this monorepo package.json would resolve it for entire app.
You'll notice that electronjs based resolutions are restricted to [ui/desktop/electron-app/package.json](https://github.com/hashicorp/boundary-ui/blob/889461770985f316507a65a7bf232d828637adc5/ui/desktop/electron-app/package.json#L44) as it's not shared across this monorepo.

Reply:
Cool. Thank you for explaining that for me.

Reply:
Thank you @gsusmi for the clarification.

@cameronperera I am not sure if you noticed, but we declare these dependencies as `resolutions`, it is a yarn specific feature, [more info about it here](https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/).

In a nutshell, let's say we have `dependencyA` that has a sub-dependency in `dependencyB@1.0.0`.

If we add `dependencyB@1.5.0` within resolutions, `dependencyA` will resolve version `1.5.0` of `dependencyB`.

We use this to be compliant with security vulnerabilities.

Reply:
I did not notice it was in `resolutions`. Thanks for pointing that out, makes perfect sense now.